### PR TITLE
My first commit. Just getting started.

### PR DESCRIPTION
**Pull request by Greyflanel**

### CODE REVIEW

File: components/Tabs/Tabs.js [hunk 1]
```diff
@@ -1,61 +1,63 @@
...
-    // const cards = ;
-
-    // Iterate through the NodeList setting the display style each one to 'none'
-    // cards.forEach()
+     const cards = document.querySelectorAll('.card');
+    //console.log(cards);
     
+    // Iterate through the NodeList setting the display style each one to 'none'
+     cards.forEach(eachCard => eachCard.classList.add(eachCard.style.display = 'none'));
     // Add a class of ".active-tab" to this.tabElement
```
Comment (wcolts2000) [inline components/Tabs/Tabs.js]: I am actually fascinated that this line is working and not throwing an error. the classList.add() method is expecting to receive a string with a class name to add to the elements classes. this line is indeed adding a class of "none" to every card element in the document if you inspect it and look at the DOM elements. they all have a class of none on them. the line of code you have inside the method where you are modifying the display style is also running and adding the inline style of display none to the card elements as well so the ending result is still the desired result even though it was achieved in a way that looks like it should throw an error. I am really intrigued here

File: components/Tabs/Tabs.js [hunk 2]
```diff
@@ -1,61 +1,63 @@
...
-    // this.tabElement;
+     this.tabElement.classList.add('active-tab');
```
Comment (wcolts2000) [inline components/Tabs/Tabs.js]: note how you add a class to the classlist here

File: components/Tabs/Tabs.js [hunk 3]
```diff
@@ -1,61 +1,63 @@
...
   constructor(cardElement){
     // Assign this.cardElement to the cardElement DOM reference
-    // this.cardElement;
+     this.cardElement = cardElement;
   }
   selectCard(){
     // Update the style of this.cardElement to display = "flex"
-    // this.cardElement;
+     this.cardElement.style.display = 'flex';
   }
```
Comment (wcolts2000) [inline components/Tabs/Tabs.js]: and note how you add a new display property to the style here

File: index.html [hunk 4]
```diff
@@ -30,7 +30,7 @@
 
   <!-- HEADER COMPONENT -->
   <div class="header">
-    <span class="date">SMARCH 28, 2018</span>
+    <span class="date">MARCH 28, 2018</span>
     <h1>Lambda Times</h1>
```
Comment (wcolts2000) [inline index.html]: great job with attention to detail here and removing the s in there. most miss that

File: components/Tabs/Tabs.js (hunk 1 above)
Comment (Greyflanel) [reply components/Tabs/Tabs.js]: That's crazy. This is actually not the first time that I have written something that shouldn't work, but does. I don't know if it's a curse or a blessing☺ I look forward to hearing your thoughts on this.

**Pull request merged**